src/components/Tables/Events.tsx

```diff
@@ -1,84 +1,170 @@
+        (event: EventWithArtistVenue) => {
+            setFeaturedMutation(
+                { id: event.id },
+                {
+                    onSuccess: () => {
+                        void refetch()
+                    },
+                    onError: (e) => {
+                        setError(
+                            'Setting featured event failed. Please try again later.'
+                        )
+                        console.error(e)
+                    }
+                }
+            )
+        },
+        [setFeaturedMutation, refetch]
+    )
+
+    const handleDelete = useCallback(
+        (event: EventWithArtistVenue) => {
+            deleteMutation(
+                { id: event.id },
+                {
+                    onSuccess: () => {
+                        void refetch()
+                    },
+                    onError: (e) => {
+                        setError('Delete venue failed. Please try again later.')
+                        console.error(e)
+                    }
+                }
+            )
+        },
+        [deleteMutation, refetch]
+    )
+
+    const columns = useMemo(
         () => [
-            {
-                accessorKey: 'name',
+            columnHelper.accessor((row) => row.name, {
                 cell: (info) => info.getValue(),
-                header: () => <span>Event</span>
-            },
-            {
-                accessorKey: 'venue.name',
+                header: 'Event'
+            }),
+            columnHelper.accessor((row) => row.venue.name, {
                 cell: (info) => info.getValue(),
-                header: () => <span>Venue</span>
-            },
-            {
-                accessorKey: 'startDate',
-                cell: (info) =>
-                    format(new Date(info.getValue() as string), 'h:mm a'),
-                header: () => <span>Start</span>,
+                header: 'Venue'
+            }),
+            columnHelper.accessor((row) => row.startDate, {
+                cell: (info) => {
+                    const content = info.getValue()
+                    if (content) {
+                        format(
+                            new Date(content as unknown as string),
+                            'h:mm a'
+                        )
+                    }
+                },
+                header: 'Start',
                 enableColumnFilter: false
-            },
-            {
-                accessorKey: 'endDate',
-                cell: (info) =>
-                    format(new Date(info.getValue() as string), 'h:mm a'),
-                header: () => <span>End</span>,
+            }),
+            columnHelper.accessor((row) => row.endDate, {
+                cell: (info) => {
+                    const content = info.getValue()
+                    if (content) {
+                        format(
+                            new Date(content as unknown as string),
```

Comment:
This still needs to be fixed. We should never have to alias like this.

Reply:
OoOOh kay I understand now. I was a bit confused exactly what you meant.